File: hw8/Facts.java
/* print square roots in Java language.  R. Brown, 9/2010 */

class Facts {
    public static long factorial(int n){
	long ans;
	ans = 1;
	for(int i = 1; i <=n; i++)
	    ans = ans*i;
	return ans;
    }
  public static void main(String args[]) {
    System.out.println("n\tn!");
    System.out.println("----------------");
    int n;
    for (n=0;  n<=5;  n++)
	System.out.println(n + "\t" + Facts.factorial(n));
    return;

  }
}

File: hw4/facts.py
# print square roots in Python language.  R. Brown, 9/2

print("n\tn!\n----------------")

def factorial(n):
    if n==0:
        return 1
    else:
        return n*factorial(n-1)

for n in range(0, 6):
    print(n,'\t', factorial(n))
